kolibri/plugins/coach/assets/src/views/exams/CreateExamPage/CreateExamPreview.vue

```diff
@@ -0,0 +1,333 @@
+<template>
```

Comment:
There's a "Preview exam" label on the app bar that seems to be missing? @indirectlylit